.ci/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -l
+
+if [ -z "$DEBUG" ]
+then
+    set -x
+fi
+
+echo "DEBUG: {SRC_ROOT} = ${SRC_ROOT}"
+
+module load hpcx-sharpv2-gcc
+
+echo "DEBUG: CUDA_HOME = ${CUDA_HOME}"
+echo "DEBUG: HPCX_SHARP_DIR = ${HPCX_SHARP_DIR}"
+
+cd ${SRC_ROOT}
+
+${SRC_ROOT}/autogen.sh
+if [ $? -ne 0 ]
+then
+    echo "ERROR: ${SRC_ROOT}/autogen.sh failed"
+    echo "FAIL"
+    exit 1
+fi
+
+${SRC_ROOT}/configure \
+    --prefix=${SRC_ROOT}/_install \
+    --with-cuda=${CUDA_HOME} \
+    --with-sharp=${HPCX_SHARP_DIR}
+if [ $? -ne 0 ]
+then
+    echo "ERROR: ${SRC_ROOT}/configure failed"
+    echo "FAIL"
+    exit 1
+fi
+
+make -j install
```

Comment:
it will take all available CPU`s
best to divide ncpus/n-executors to mind other executors.

Reply:
For this an agreement on CPU splitting policies are needed and other jobs should follow to them. I can add the policies already used in UCX:
`AFFINITY="taskset -c $(( 2 * EXECUTOR_NUMBER ))","$(( 2 * EXECUTOR_NUMBER + 1))"`

Reply:
I'm thinking about runtime check for number of available jenkins executors on a slave and calculating taskset/affinity mask for the job scripts based on that. Need to sync with MichaelB for this (seems some permissions for groovy pipeline scripts may be required). I'd recommend to consider this as a future improvement (need some time to investigate how to implement this - I'll put this into the TODO section into Jenkinsfile). Right now there's only 1 executor is configured on target slaves.